src/networks.json

```diff
@@ -1246,6 +1246,23 @@
     "start": 3673983,
     "logo": "ipfs://QmUkFZC2ZmoYPTKf7AHdjwRPZoV2h1MCuHaGM4iu8SNFpi"
   },
+  "7700": {
+    "key": "7700",
+    "name": "Canto",
+    "shortName": "Canto",
+    "chainId": 7700,
+    "network": "mainnet",
+    "testnet": false,
+    "multicall": "0x9787792A1020B00fb65191754c3D87c89Bf36444",
+    "rpc": [
+      "https://canto.neobase.one"
+    ],
+    "explorer": {
+      "url": "https://evm.explorer.canto.io"
+    },
+    "start": 1683285,
+    "logo": "ipfs://QmYiL4FApnrLePTmGktVE3prmLBubCiFBfUKFpgX32jaaZ"
```

Comment:
Hi @akshatcx Looks like the logo is not pinned, it can't resolve `QmYiL4FApnrLePTmGktVE3prmLBubCiFBfUKFpgX32jaaZ`
Can you try to pin it to a third-party pining service? everything else looks good

Reply:
Yes, it seems to have been removed. I have updated it with a new one which is pinned using infura. 

https://ipfs.io/ipfs/QmVh5XRi7xN4UNw7SNKpEhjF1TsPXJnRyNkB4EuMVTVCB6